test/Table_Tests/src/Util.enso

```diff
@@ -13,9 +14,9 @@ Table.should_equal self expected =
 
 Column.should_equal self expected =
     if self.name != expected.name then
-        Test.fail "Expected column name "+expected.name+", but got "+self.name+"."
+        fail "Expected column name "+expected.name+", but got "+self.name+"."
```

Comment:
Are you sure?

I liked the way this used to be (quite subjective though, so feel free to ignore) - `Test.fail` makes it clear to me that we are calling a method to fail the test and makes it relatively easy to locate where it was declared.

Just `fail` makes it easier to be overridden by some local definition and is a bit less readable as to what kind of failure that is.

Reply:
Also I think we should be consistent - in some other places like `Range_Spec` we still use `Test.fail` and I think this nicely indicates what this is.

I'm fine with using just shorter `fail` inside of `Extensions.enso` within the `Test` library since there we are in the "context" of the Test library, so that seems fine.

But if possible, outside I'd prefer `Test.fail`. Of course open to discussion though!